masterlist.yaml

```
@@ -26918,4 +26943,4 @@ plugins:
       - crc: 0xFC4C5AEF
         util: 'SSEEdit v4.0.4c'
       - crc: 0xE405E000
-        util: 'SSEEdit v4.0.4c'
+        util: 'SSEEdit v4.0.4c'
```

The ending newline was unintentionally removed. Looks good otherwise:
```suggestion
        util: 'SSEEdit v4.0.4c'

```